tests/test_utils/test_apply_theme.py

```
@@ -0,0 +1,26 @@
+import pytest
```

Ruff (0.8.2)

File `tests/test_utils/test_apply_theme.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)